package.json

```diff
@@ -32,72 +32,72 @@
     "babelify": "^7.2.0",
     "body-parser": "^1.15.2",
     "browser-sync": "^2.11.1",
-    "browserify": "^12.0.1",
-    "chalk": "^1.1.3",
-    "client-sessions": "^0.7.0",
+    "browserify": "^14.5.0",
+    "chalk": "^2.2.0",
+    "client-sessions": "^0.8.0",
     "compression": "^1.6.1",
     "cors": "^2.8.1",
     "country-list": "^1.0.0",
-    "css-loader": "^0.21.0",
-    "del": "^2.0.2",
+    "css-loader": "^0.28.7",
+    "del": "^3.0.0",
     "dotenv": "^4.0.0",
     "express": "^4.13.3",
     "express-session": "^1.14.2",
-    "extract-text-webpack-plugin": "^0.9.0",
+    "extract-text-webpack-plugin": "^3.0.1",
     "forms": "https://github.com/hackcambridge/forms/tarball/master",
-    "google-spreadsheet": "^1.1.3",
-    "gulp": "^3.9.0",
-    "gulp-autoprefixer": "^3.1.0",
+    "google-spreadsheet": "^2.0.4",
+    "gulp": "*",
+    "gulp-autoprefixer": "^4.0.0",
     "gulp-if": "^2.0.0",
     "gulp-load-plugins": "^1.0.0",
     "gulp-plumber": "^1.0.1",
-    "gulp-rev-all": "^0.8.22",
-    "gulp-sourcemaps": "^1.6.0",
+    "gulp-rev-all": "^0.9.7",
+    "gulp-sourcemaps": "^2.6.1",
     "gulp-stylus": "^2.1.0",
-    "gulp-uglify": "^1.4.2",
+    "gulp-uglify": "^3.0.0",
     "gulp-util": "^3.0.7",
     "gutils": "0.0.5",
-    "jquery": "^2.1.4",
+    "jquery": "^3.2.1",
     "js-yaml": "^3.4.3",
-    "lodash": "^3.10.1",
+    "lodash": "^4.17.4",
     "mailchimp-api": "^2.0.7",
-    "mailgen": "2.0.1",
-    "markdown-it": "^5.1.0",
-    "markdown-it-attrs": "^0.1.8",
+    "mailgen": "2.0.2",
+    "markdown-it": "^8.4.0",
+    "markdown-it-attrs": "^1.2.0",
     "moment": "^2.11.1",
     "moment-timezone": "^0.5.0",
     "multer": "^1.2.0",
     "multer-s3": "^2.3.2",
     "node-fetch": "^1.6.3",
-    "nodemailer": "^2.6.4",
+    "nodemailer": "^4.2.0",
     "nodemailer-mailgun-transport": "^1.2.2",
     "nodemailer-sendmail-transport": "^1.0.0",
     "nodemon": "^1.8.1",
-    "normalize.css": "^3.0.3",
-    "nunjucks": "^2.1.0",
-    "passport": "^0.3.2",
+    "normalize.css": "^7.0.0",
+    "nunjucks": "^3.0.1",
+    "passport": "^0.4.0",
     "passport-http-bearer": "^1.0.1",
     "pepjs": "^0.4.1",
-    "pg": "^6.1.0",
+    "pg": "^7.3.0",
     "pg-hstore": "^2.3.2",
     "pusher-js": "^4.0.0",
     "querystring": "^0.2.0",
-    "run-sequence": "^1.1.4",
-    "sequelize": "^3.24.8",
-    "socket.io": "^1.4.5",
-    "socket.io-client": "^1.4.5",
-    "stripe": "^4.1.0",
-    "style-loader": "^0.13.0",
-    "stylus-loader": "^1.4.2",
+    "run-sequence": "^2.2.0",
+    "sequelize": "^4.15.2",
```

Comment:
Running `yarn run migrate` on a fresh repository to attempt to initialise the database is no longer working for me, I suspect this might be a result of upgrading sequelize.

Reply:
will look into it